pulp_container/app/token_verification.py

```diff
@@ -8,49 +8,60 @@
 from rest_framework.permissions import BasePermission, SAFE_METHODS
 
 
-def _decode_token(encoded_token, request):
```

Comment:
My comments on the old PR for this file are still valid.
I think the factory pattern, to compare three strings here, is adding an overwhelming amount of complexity.